### PR TITLE
Update voting members and add Kris Borchers to regular members

### PR DESCRIPTION
**Pull request by kborchers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,11 +85,8 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
 
 According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the five individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.
```
Comment (tobie) [inline README.md:86]:
```suggestion
According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole.
```

File: README.md [hunk 2]
```diff
@@ -85,11 +85,8 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
...
 
-- **Architect**: Kris Borchers (@kborchers)
-- **messageformat**: Eemeli Aro (@eemeli)
+- **AMP**: Tobie Langel (@tobie)
 - **Mocha**: Christopher Hiller (@boneskull)
```
Comment (tobie) [inline README.md:89]:
```suggestion
- Christopher Hiller (@boneskull)
```
Given we're representing the group of projects as a whole, I think it doesn't make sense for our affiliation to be noted.

File: README.md [hunk 3]
```diff
@@ -85,11 +85,8 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
 
 According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the five individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.
 
-- **Architect**: Kris Borchers (@kborchers)
-- **messageformat**: Eemeli Aro (@eemeli)
+- **AMP**: Tobie Langel (@tobie)
```
Comment (tobie) [inline README.md:88]:
```suggestion
- Tobie Langel (@tobie)
```

**Comment by kborchers** [comment]
Closing in favor of #519

**Pull request closed without merging**